### PR TITLE
rename GreetService to HappenedService

### PR DESCRIPTION
**Pull request by anmho**
Rename GreetService to HappenedService
Update Makefile with QoL commands

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,13 +2,22 @@
 
 .PHONY: deps
 deps:
-	@go install github.com/air-verse/air@latest
+	@go install "github.com/bufbuild/buf/cmd/buf@latest"
```
Comment (anmho) [inline Makefile:5]: deps target for easy setup

Comment (anmho) [reply Makefile:5]: Note: should probably also include golang install?

File: Makefile [hunk 2]
```diff
@@ -2,13 +2,22 @@
...
 
 .PHONY: build
 build:
 	@cd api && go build -o ./bin/api ./cmd/api
+
 .PHONY: watch
 watch:
 	@air --build.cmd "make build" --build.bin "./api/bin/api"
 
+.PHONY: gen
```
Comment (anmho) [inline Makefile:20]: Command for buf gen shortcut

File: api/cmd/api/main.go [hunk 3]
```diff
@@ -7,36 +7,41 @@ import (
...
+	"happenedapi/gen/protos/v1/happenedv1connect"
 	"log"
 	"log/slog"
 	"net/http"
 	"os"
 )
 
 // Ensure interface satisfaction
-var _ greetv1connect.GreetServiceHandler = (*GreetServer)(nil)
+var _ happenedv1connect.HappenedServiceHandler = (*HappenedServer)(nil)
```
Comment (anmho) [inline api/cmd/api/main.go:19]: Renaming to HappenedService

**Review by Jish2 — APPROVED** [review]
lgtm. could you add proto js outputs to makefile

**Pull request merged**